### ISSUE TITLE
Increase speed, show progress bar

### ISSUE THREAD

**Issue by daohu527** [daohu527/cyber_record#64]
Increase speed, show progress bar

**Comment by dufengit** [comment]
Any progress for this, read_messages is really slow when read message, is there faster solutions to read log?

**Comment by daohu527** [comment]
Yes, you can use [filter](https://github.com/daohu527/cyber_record?tab=readme-ov-file#filter-read) to increase speed
